src/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ ADD_SUBDIRECTORY(drm)
 ADD_SUBDIRECTORY(dstripe)
 ADD_SUBDIRECTORY(dsync)
 IF(ENABLE_DAOS AND ENABLE_HDF5)
+  ADD_SUBDIRECTORY(daos-gen)
   ADD_SUBDIRECTORY(daos-serialize)
   ADD_SUBDIRECTORY(daos-deserialize)
```

Comment:
@dsikich I know you're using this for testing serialization, but does `daos-gen` itself require HDF5? If not, maybe we don't need to enforce that?